### ISSUE TITLE
publish a new release on pypi?

### ISSUE THREAD

**Issue by bdewilde** [seomoz/dragnet#60]
Given the huge number of changes to `dragnet` since the last release (v1.1, Jan '17), are there near-term plans to publish a new release on PyPi? I see that the failing test got removed, but don't know if there are other critical to-do's before everything's release-ready.

**Comment by pakelley** [comment]
I'd like to get up those models I said I'd train in #52 (thinking I'll have that done today, but maybe tomorrow), but aside from that there's nothing I can think of that needs to be done before a release

**Comment by pakelley** [comment]
@bdewilde , I'm using the GridSearchCV you added in to try and get some decent params for these models, but when I try to run the grid search with more than one worker I get an error about not being able to pickle the `lxml` Element objects that are passed into `fit`. (I can copy in a stack trace, etc. if it's helpful)

From what I can tell, this is expected behavior for multiprocessing with `joblib`, but do you know of a way around this? Do you remember this being an issue for you when you were working on your changes?

btw, thanks a ton for all your contribution here!

**Comment by bdewilde** [comment]
Glad that the updated model training code is being put to good use! Unfortunately, I don't have a solution for this problem — I never tried training in parallel (`n_jobs > 1`, right?), and instead just bit the bullet on training time. Just checking: Is the latter option prohibitively slow?

**Comment by pakelley** [comment]
Yeah exactly, `n_jobs`>1, but no it's not terribly slow, just wondering if there's an obvious solution I was missing. I'll get some models trained and uploaded asap!

One more thing on the models: would it make sense to use LFS to commit them?

**Comment by bdewilde** [comment]
re: git-lfs: I'm not sure. How large are the pickled models? I remember the updated ones being _larger_ than the old ones, but not the absolute values. If the files are > 100MB or so, I _think_ git-lfs may be a good idea, but since we won't be modifying these models often, it probably won't make a huge difference in developers' lives.

**Comment by pakelley** [comment]
The models are only 40-50MB, so that seems fine to commit them normally. I wasn't sure what a good threshold for that was, but if it seems good to everyone working on this it's good with me.

**Comment by nehalecky** [comment]
@bdewilde / @b4hand. With #63 and #65 merged, we're now good with a new release of dragnet being published to pypi. I recall a discussion of someone around here having access to the [dragnet pypi account](https://pypi.python.org/pypi/dragnet), but don't recall who specifically. Any help here would be appreciated, thanks! And thanks @pakelley for the effort getting the python 3 port to be release-ready!

**Comment by b4hand** [comment]
I currently don't have access to the pypi release for that project, but it appears that @dlecocq and @matt-peters do. Could one of you grant me access?

**Comment by dlecocq** [comment]
That'll have to be @matt-peters - it doesn't appear that I have access, either.

**Comment by nehalecky** [comment]
thanks for the replies, all. @matt-peters, let us know if we can help. We look forward to this so we can streamline our dependencies!  Also, FWIW, we're also interested in maintaining a conda package per release, and will be submitting a pull request for that in the near future. :)

**Comment by matt-peters** [comment]
OK, new version (2.0.2) is available on via pip. Check it out and let me know if it you run into problems.

**Comment by b4hand** [comment]
It looks like the version in GitHub wasn't actually bumped in `setup.py`. I'll go ahead and update that as well.

**Comment by nehalecky** [comment]
hey @b4hand and @matt-peters, great to hear! FWIW, the changes made in #63 #52 we're substantial, including breaking external API changes. I don't think we mentioned it specifically, however, with semver, you're likely going to want to do a major version bump! Dragnet 3.0.0, now supporting Python 3! :)

**Comment by b4hand** [comment]
Prior version was 1.1.0, so 2.0.2 is a major version bump already. I think @matt-peters had an issue with the initial 2.0 release, so that's why it is 2.0.2.

**Comment by nehalecky** [comment]
OHHHH, my bad. pip wasn't super helpful in getting a version history. THANKS MUCH

**Comment by matt-peters** [comment]
What @b4hand said.  The first attempts were broken (needed a custom `MANIFEST.in` to include the cython files).

**Comment by b4hand** [comment]
Version updated in 5b8855319df7a9e6b114094046b10da4ed8c9e9b.

**Comment by rajkunkolienkar** [comment]
I am using python 3.6, unable to install via pip.
```
Collecting dragnet`
  Using cached https://files.pythonhosted.org/packages/d8/5b/b1ea2d21c45dc6862b950530e96bf9c7db0a20dd313384ae8aac4140e41a/dragnet-2.0.2.tar.gz
Requirement already satisfied: Cython>=0.21.1 in ./lib64/python3.6/site-packages (from dragnet) (0.28.2)
Requirement already satisfied: lxml in ./lib64/python3.6/site-packages (from dragnet) (4.2.1)
Collecting scikit-learn<0.19.0,>=0.15.2 (from dragnet)
  Downloading https://files.pythonhosted.org/packages/d1/51/ca5be39c576c981cf8b8359fcb1ee49ca43d59b833415205cec5ef5c0fcd/scikit_learn-0.18.2-cp36-cp36m-manylinux1_x86_64.whl (11.8MB)
    100% |████████████████████████████████| 11.8MB 4.8MB/s
Requirement already satisfied: numpy in ./lib64/python3.6/site-packages (from dragnet) (1.14.2)
Collecting scipy (from dragnet)
  Cache entry deserialization failed, entry ignored
  Downloading https://files.pythonhosted.org/packages/2c/13/eb888fcc83f14d114dee794c3491477ce156caa9f456b7bef1112dde36b5/scipy-1.0.1-cp36-cp36m-manylinux1_x86_64.whl (50.0MB)
    100% |████████████████████████████████| 50.0MB 778kB/s
Requirement already satisfied: ftfy<5.0.0,>=4.1.0 in ./lib/python3.6/site-packages (from dragnet) (4.4.3)
Requirement already satisfied: html5lib in ./lib/python3.6/site-packages (from ftfy<5.0.0,>=4.1.0->dragnet) (1.0.1)
Requirement already satisfied: wcwidth in ./lib/python3.6/site-packages (from ftfy<5.0.0,>=4.1.0->dragnet) (0.1.7)
Requirement already satisfied: webencodings in ./lib/python3.6/site-packages (from html5lib->ftfy<5.0.0,>=4.1.0->dragnet) (0.5.1)
Requirement already satisfied: six>=1.9 in ./lib/python3.6/site-packages (from html5lib->ftfy<5.0.0,>=4.1.0->dragnet) (1.11.0)
Building wheels for collected packages: dragnet
  Running setup.py bdist_wheel for dragnet ... error
  Complete output from command /home/ec2-user/lambdas/content_extractor/dragnet/bin/python36 -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-ebpe6z1l/dragnet/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" bdist_wheel -d /tmp/pip-wheel-qeif6f4a --python-tag cp36:
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build/lib.linux-x86_64-3.6
  creating build/lib.linux-x86_64-3.6/dragnet
  copying dragnet/model_training.py -> build/lib.linux-x86_64-3.6/dragnet
  copying dragnet/data_processing.py -> build/lib.linux-x86_64-3.6/dragnet
  copying dragnet/__init__.py -> build/lib.linux-x86_64-3.6/dragnet
  copying dragnet/extractor.py -> build/lib.linux-x86_64-3.6/dragnet
  copying dragnet/compat.py -> build/lib.linux-x86_64-3.6/dragnet
  copying dragnet/util.py -> build/lib.linux-x86_64-3.6/dragnet
  creating build/lib.linux-x86_64-3.6/dragnet/features
  copying dragnet/features/css.py -> build/lib.linux-x86_64-3.6/dragnet/features
  copying dragnet/features/standardized.py -> build/lib.linux-x86_64-3.6/dragnet/features
  copying dragnet/features/kohlschuetter.py -> build/lib.linux-x86_64-3.6/dragnet/features
  copying dragnet/features/readability.py -> build/lib.linux-x86_64-3.6/dragnet/features
  copying dragnet/features/weninger.py -> build/lib.linux-x86_64-3.6/dragnet/features
  copying dragnet/features/__init__.py -> build/lib.linux-x86_64-3.6/dragnet/features
  creating build/lib.linux-x86_64-3.6/dragnet/pickled_models
  creating build/lib.linux-x86_64-3.6/dragnet/pickled_models/sklearn_0.18.0
  copying dragnet/pickled_models/sklearn_0.18.0/kohlschuetter_weninger_readability_content_comments_model.pickle.gz -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/sklearn_0.18.0
  copying dragnet/pickled_models/sklearn_0.18.0/kohlschuetter_weninger_readability_content_model.pickle.gz -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/sklearn_0.18.0
  creating build/lib.linux-x86_64-3.6/dragnet/pickled_models/py2_sklearn_0.18.0
  copying dragnet/pickled_models/py2_sklearn_0.18.0/kohlschuetter_readability_weninger_comments_content_model.pkl.gz -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py2_sklearn_0.18.0
  copying dragnet/pickled_models/py2_sklearn_0.18.0/kohlschuetter_readability_weninger_comments_content_block_errors.txt -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py2_sklearn_0.18.0
  copying dragnet/pickled_models/py2_sklearn_0.18.0/kohlschuetter_readability_weninger_content_block_errors.txt -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py2_sklearn_0.18.0
  copying dragnet/pickled_models/py2_sklearn_0.18.0/kohlschuetter_readability_weninger_comments_block_errors.txt -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py2_sklearn_0.18.0
  copying dragnet/pickled_models/py2_sklearn_0.18.0/kohlschuetter_readability_weninger_comments_model.pkl.gz -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py2_sklearn_0.18.0
  copying dragnet/pickled_models/py2_sklearn_0.18.0/kohlschuetter_readability_weninger_content_model.pkl.gz -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py2_sklearn_0.18.0
  creating build/lib.linux-x86_64-3.6/dragnet/pickled_models/py2_sklearn_0.15.2_0.17.1
  copying dragnet/pickled_models/py2_sklearn_0.15.2_0.17.1/kohlschuetter_readability_weninger_comments_content_model.pkl.gz -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py2_sklearn_0.15.2_0.17.1
  copying dragnet/pickled_models/py2_sklearn_0.15.2_0.17.1/kohlschuetter_readability_weninger_comments_content_block_errors.txt -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py2_sklearn_0.15.2_0.17.1
  copying dragnet/pickled_models/py2_sklearn_0.15.2_0.17.1/kohlschuetter_readability_weninger_content_block_errors.txt -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py2_sklearn_0.15.2_0.17.1
  copying dragnet/pickled_models/py2_sklearn_0.15.2_0.17.1/kohlschuetter_readability_weninger_comments_block_errors.txt -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py2_sklearn_0.15.2_0.17.1
  copying dragnet/pickled_models/py2_sklearn_0.15.2_0.17.1/kohlschuetter_readability_weninger_comments_model.pkl.gz -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py2_sklearn_0.15.2_0.17.1
  copying dragnet/pickled_models/py2_sklearn_0.15.2_0.17.1/kohlschuetter_readability_weninger_content_model.pkl.gz -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py2_sklearn_0.15.2_0.17.1
  creating build/lib.linux-x86_64-3.6/dragnet/pickled_models/sklearn_0.15.2_0.17.1
  copying dragnet/pickled_models/sklearn_0.15.2_0.17.1/kohlschuetter_weninger_readability_content_comments_model.pickle.gz -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/sklearn_0.15.2_0.17.1
  copying dragnet/pickled_models/sklearn_0.15.2_0.17.1/kohlschuetter_weninger_readability_content_model.pickle.gz -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/sklearn_0.15.2_0.17.1
  creating build/lib.linux-x86_64-3.6/dragnet/pickled_models/py3_sklearn_0.15.2_0.17.1
  copying dragnet/pickled_models/py3_sklearn_0.15.2_0.17.1/kohlschuetter_readability_weninger_comments_content_model.pkl.gz -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py3_sklearn_0.15.2_0.17.1
  copying dragnet/pickled_models/py3_sklearn_0.15.2_0.17.1/kohlschuetter_readability_weninger_comments_content_block_errors.txt -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py3_sklearn_0.15.2_0.17.1
  copying dragnet/pickled_models/py3_sklearn_0.15.2_0.17.1/kohlschuetter_readability_weninger_content_block_errors.txt -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py3_sklearn_0.15.2_0.17.1
  copying dragnet/pickled_models/py3_sklearn_0.15.2_0.17.1/kohlschuetter_readability_weninger_comments_block_errors.txt -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py3_sklearn_0.15.2_0.17.1
  copying dragnet/pickled_models/py3_sklearn_0.15.2_0.17.1/kohlschuetter_readability_weninger_comments_model.pkl.gz -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py3_sklearn_0.15.2_0.17.1
  copying dragnet/pickled_models/py3_sklearn_0.15.2_0.17.1/kohlschuetter_readability_weninger_content_model.pkl.gz -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py3_sklearn_0.15.2_0.17.1
  creating build/lib.linux-x86_64-3.6/dragnet/pickled_models/py3_sklearn_0.18.0
  copying dragnet/pickled_models/py3_sklearn_0.18.0/kohlschuetter_readability_weninger_comments_content_model.pkl.gz -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py3_sklearn_0.18.0
  copying dragnet/pickled_models/py3_sklearn_0.18.0/kohlschuetter_readability_weninger_comments_content_block_errors.txt -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py3_sklearn_0.18.0
  copying dragnet/pickled_models/py3_sklearn_0.18.0/kohlschuetter_readability_weninger_content_block_errors.txt -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py3_sklearn_0.18.0
  copying dragnet/pickled_models/py3_sklearn_0.18.0/kohlschuetter_readability_weninger_comments_block_errors.txt -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py3_sklearn_0.18.0
  copying dragnet/pickled_models/py3_sklearn_0.18.0/kohlschuetter_readability_weninger_comments_model.pkl.gz -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py3_sklearn_0.18.0
  copying dragnet/pickled_models/py3_sklearn_0.18.0/kohlschuetter_readability_weninger_content_model.pkl.gz -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py3_sklearn_0.18.0
  running build_ext
  building 'dragnet.lcs' extension
  creating build/temp.linux-x86_64-3.6
  creating build/temp.linux-x86_64-3.6/dragnet
  gcc -pthread -Wno-unused-result -Wsign-compare -DDYNAMIC_ANNOTATIONS_ENABLED=1 -DNDEBUG -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic -D_GNU_SOURCE -fPIC -fwrapv -fPIC -I/home/ec2-user/lambdas/content_extractor/dragnet/local/lib64/python3.6/site-packages/numpy/core/include -I/usr/include/python3.6m -c dragnet/lcs.cpp -o build/temp.linux-x86_64-3.6/dragnet/lcs.o
  unable to execute 'gcc': No such file or directory
  error: command 'gcc' failed with exit status 1

  ----------------------------------------
  Failed building wheel for dragnet
  Running setup.py clean for dragnet
Failed to build dragnet
Installing collected packages: scikit-learn, scipy, dragnet
  Found existing installation: scikit-learn 0.19.0
    Uninstalling scikit-learn-0.19.0:
      Successfully uninstalled scikit-learn-0.19.0
  Running setup.py install for dragnet ... error
    Complete output from command /home/ec2-user/lambdas/content_extractor/dragnet/bin/python36 -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-ebpe6z1l/dragnet/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-record-nj9w5qc1/install-record.txt --single-version-externally-managed --compile --install-headers /home/ec2-user/lambdas/content_extractor/dragnet/include/site/python3.6/dragnet:
    running install
    running build
    running build_py
    creating build
    creating build/lib.linux-x86_64-3.6
    creating build/lib.linux-x86_64-3.6/dragnet
    copying dragnet/model_training.py -> build/lib.linux-x86_64-3.6/dragnet
    copying dragnet/data_processing.py -> build/lib.linux-x86_64-3.6/dragnet
    copying dragnet/__init__.py -> build/lib.linux-x86_64-3.6/dragnet
    copying dragnet/extractor.py -> build/lib.linux-x86_64-3.6/dragnet
    copying dragnet/compat.py -> build/lib.linux-x86_64-3.6/dragnet
    copying dragnet/util.py -> build/lib.linux-x86_64-3.6/dragnet
    creating build/lib.linux-x86_64-3.6/dragnet/features
    copying dragnet/features/css.py -> build/lib.linux-x86_64-3.6/dragnet/features
    copying dragnet/features/standardized.py -> build/lib.linux-x86_64-3.6/dragnet/features
    copying dragnet/features/kohlschuetter.py -> build/lib.linux-x86_64-3.6/dragnet/features
    copying dragnet/features/readability.py -> build/lib.linux-x86_64-3.6/dragnet/features
    copying dragnet/features/weninger.py -> build/lib.linux-x86_64-3.6/dragnet/features
    copying dragnet/features/__init__.py -> build/lib.linux-x86_64-3.6/dragnet/features
    creating build/lib.linux-x86_64-3.6/dragnet/pickled_models
    creating build/lib.linux-x86_64-3.6/dragnet/pickled_models/sklearn_0.18.0
    copying dragnet/pickled_models/sklearn_0.18.0/kohlschuetter_weninger_readability_content_comments_model.pickle.gz -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/sklearn_0.18.0
    copying dragnet/pickled_models/sklearn_0.18.0/kohlschuetter_weninger_readability_content_model.pickle.gz -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/sklearn_0.18.0
    creating build/lib.linux-x86_64-3.6/dragnet/pickled_models/py2_sklearn_0.18.0
    copying dragnet/pickled_models/py2_sklearn_0.18.0/kohlschuetter_readability_weninger_comments_content_model.pkl.gz -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py2_sklearn_0.18.0
    copying dragnet/pickled_models/py2_sklearn_0.18.0/kohlschuetter_readability_weninger_comments_content_block_errors.txt -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py2_sklearn_0.18.0
    copying dragnet/pickled_models/py2_sklearn_0.18.0/kohlschuetter_readability_weninger_content_block_errors.txt -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py2_sklearn_0.18.0
    copying dragnet/pickled_models/py2_sklearn_0.18.0/kohlschuetter_readability_weninger_comments_block_errors.txt -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py2_sklearn_0.18.0
    copying dragnet/pickled_models/py2_sklearn_0.18.0/kohlschuetter_readability_weninger_comments_model.pkl.gz -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py2_sklearn_0.18.0
    copying dragnet/pickled_models/py2_sklearn_0.18.0/kohlschuetter_readability_weninger_content_model.pkl.gz -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py2_sklearn_0.18.0
    creating build/lib.linux-x86_64-3.6/dragnet/pickled_models/py2_sklearn_0.15.2_0.17.1
    copying dragnet/pickled_models/py2_sklearn_0.15.2_0.17.1/kohlschuetter_readability_weninger_comments_content_model.pkl.gz -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py2_sklearn_0.15.2_0.17.1
    copying dragnet/pickled_models/py2_sklearn_0.15.2_0.17.1/kohlschuetter_readability_weninger_comments_content_block_errors.txt -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py2_sklearn_0.15.2_0.17.1
    copying dragnet/pickled_models/py2_sklearn_0.15.2_0.17.1/kohlschuetter_readability_weninger_content_block_errors.txt -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py2_sklearn_0.15.2_0.17.1
    copying dragnet/pickled_models/py2_sklearn_0.15.2_0.17.1/kohlschuetter_readability_weninger_comments_block_errors.txt -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py2_sklearn_0.15.2_0.17.1
    copying dragnet/pickled_models/py2_sklearn_0.15.2_0.17.1/kohlschuetter_readability_weninger_comments_model.pkl.gz -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py2_sklearn_0.15.2_0.17.1
    copying dragnet/pickled_models/py2_sklearn_0.15.2_0.17.1/kohlschuetter_readability_weninger_content_model.pkl.gz -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py2_sklearn_0.15.2_0.17.1
    creating build/lib.linux-x86_64-3.6/dragnet/pickled_models/sklearn_0.15.2_0.17.1
    copying dragnet/pickled_models/sklearn_0.15.2_0.17.1/kohlschuetter_weninger_readability_content_comments_model.pickle.gz -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/sklearn_0.15.2_0.17.1
    copying dragnet/pickled_models/sklearn_0.15.2_0.17.1/kohlschuetter_weninger_readability_content_model.pickle.gz -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/sklearn_0.15.2_0.17.1
    creating build/lib.linux-x86_64-3.6/dragnet/pickled_models/py3_sklearn_0.15.2_0.17.1
    copying dragnet/pickled_models/py3_sklearn_0.15.2_0.17.1/kohlschuetter_readability_weninger_comments_content_model.pkl.gz -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py3_sklearn_0.15.2_0.17.1
    copying dragnet/pickled_models/py3_sklearn_0.15.2_0.17.1/kohlschuetter_readability_weninger_comments_content_block_errors.txt -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py3_sklearn_0.15.2_0.17.1
    copying dragnet/pickled_models/py3_sklearn_0.15.2_0.17.1/kohlschuetter_readability_weninger_content_block_errors.txt -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py3_sklearn_0.15.2_0.17.1
    copying dragnet/pickled_models/py3_sklearn_0.15.2_0.17.1/kohlschuetter_readability_weninger_comments_block_errors.txt -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py3_sklearn_0.15.2_0.17.1
    copying dragnet/pickled_models/py3_sklearn_0.15.2_0.17.1/kohlschuetter_readability_weninger_comments_model.pkl.gz -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py3_sklearn_0.15.2_0.17.1
    copying dragnet/pickled_models/py3_sklearn_0.15.2_0.17.1/kohlschuetter_readability_weninger_content_model.pkl.gz -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py3_sklearn_0.15.2_0.17.1
    creating build/lib.linux-x86_64-3.6/dragnet/pickled_models/py3_sklearn_0.18.0
    copying dragnet/pickled_models/py3_sklearn_0.18.0/kohlschuetter_readability_weninger_comments_content_model.pkl.gz -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py3_sklearn_0.18.0
    copying dragnet/pickled_models/py3_sklearn_0.18.0/kohlschuetter_readability_weninger_comments_content_block_errors.txt -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py3_sklearn_0.18.0
    copying dragnet/pickled_models/py3_sklearn_0.18.0/kohlschuetter_readability_weninger_content_block_errors.txt -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py3_sklearn_0.18.0
    copying dragnet/pickled_models/py3_sklearn_0.18.0/kohlschuetter_readability_weninger_comments_block_errors.txt -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py3_sklearn_0.18.0
    copying dragnet/pickled_models/py3_sklearn_0.18.0/kohlschuetter_readability_weninger_comments_model.pkl.gz -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py3_sklearn_0.18.0
    copying dragnet/pickled_models/py3_sklearn_0.18.0/kohlschuetter_readability_weninger_content_model.pkl.gz -> build/lib.linux-x86_64-3.6/dragnet/pickled_models/py3_sklearn_0.18.0
    running build_ext
    building 'dragnet.lcs' extension
    creating build/temp.linux-x86_64-3.6
    creating build/temp.linux-x86_64-3.6/dragnet
    gcc -pthread -Wno-unused-result -Wsign-compare -DDYNAMIC_ANNOTATIONS_ENABLED=1 -DNDEBUG -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector --param=ssp-buffer-size=4 -m64 -mtune=generic -D_GNU_SOURCE -fPIC -fwrapv -fPIC -I/home/ec2-user/lambdas/content_extractor/dragnet/local/lib64/python3.6/site-packages/numpy/core/include -I/usr/include/python3.6m -c dragnet/lcs.cpp -o build/temp.linux-x86_64-3.6/dragnet/lcs.o
    unable to execute 'gcc': No such file or directory
    error: command 'gcc' failed with exit status 1

    ----------------------------------------
Command "/home/ec2-user/lambdas/content_extractor/dragnet/bin/python36 -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-ebpe6z1l/dragnet/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-record-nj9w5qc1/install-record.txt --single-version-externally-managed --compile --install-headers /home/ec2-user/lambdas/content_extractor/dragnet/include/site/python3.6/dragnet" failed with error code 1 in /tmp/pip-install-ebpe6z1l/dragnet/

 Command "/home/ec2-user/lambdas/content_extractor/dragnet/bin/python36 -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-ebpe6z1l/dragnet/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-record-nj9w5qc1/install-record.txt --single-version-externally-managed --compile --install-headers /home/ec2-user/lambdas/content_extractor/dragnet/include/site/python3.6/dragnet" failed with error code 1 in /tmp/pip-install-ebpe6z1l/dragnet/
```

**Comment by b4hand** [comment]
That's unrelated to this issue. Notice the error says:
```
unable to execute 'gcc': No such file or directory
error: command 'gcc' failed with exit status 1
```
That's because you don't have `gcc` installed. `dragnet` uses several compiled extensions which means you need a compiler, specifically `gcc`. You should look at the [README installation instructions](https://github.com/seomoz/dragnet#installing-without-vagrant). Specifically, you may want to take a look at [`provision.sh`](https://github.com/seomoz/dragnet/blob/master/provision.sh) since it includes the necessary dependencies.

**Comment by b4hand** [comment]
I see you filed a separate issue at #69 as well.
